kivy/core/window/__init__.py

```diff
@@ -1078,6 +1078,14 @@ def flip(self):
         '''Flip between buffers'''
         pass
 
+    def set_window_opacity(self, opacity):
```

Comment:
This could be an `AliasProperty` named `opacity`, which would be definitely prettier than having setter and getter + it doesn't match the rest of the file. See [this](https://github.com/kivy/kivy/blob/master/kivy/core/window/__init__.py#L758).